src/main/java/controller/Game.java

```diff
@@ -0,0 +1,40 @@
+package controller;
+
+import domain.Cars;
+import domain.WinnerSelector;
+import view.View;
+
+public class Game {
+
+    public void startGame() {
+        try {
+            String inputNames = View.inputCarName();
+            int tryCount = View.inputTryCount();
+
+            Cars cars = new Cars();
+
+            //입력 받은 자동차 이름 분리 및 자동차 생성
+            String[] carNames = View.splitCarName(inputNames);
+            cars.generateCarWithNames(carNames);
```

Comment:
cars를 선언한 뒤, 필드값을 정의하는 것 보다,
필드값 정의와 함께 생성이 이루어지면 더 안정적인 코드를 작성할 수 있어요!

```suggestion
            //입력 받은 자동차 이름 분리 및 자동차 생성
            String[] carNames = View.splitCarName(inputNames);
            Cars cars = new Cars(carNames); // 새로운 생성자 작성
```